webpack.prod.config.babel.js

```diff
@@ -32,7 +32,7 @@ export default {
     // This is the path used to require files in the generated bundles.
```

Comment:
Could you add a comment above `const STATIC_DIR = ...` (on L18) that if the directory is updated, the middleware in `src/amo/middleware/staticAssets.js` should also be updated accordingly?

Reply:
Also, should we file a Jira ticket for Ops to update the script and have `staticPath` used everywhere? (instead of having `static` folder on disk and a `static-frontend` served)

Reply:
Yeah. I'll file an addons-frontend issue, I suggest we wait until after the CDN work has been deployed to update the ops script just to make coordination easier.

Reply:
https://github.com/mozilla/addons-frontend/issues/10842